src/test/java/Remy/command/MarkCommandTest.java

```diff
@@ -0,0 +1,44 @@
+package Remy.command;
+
+import Remy.ChatbotException;
+import Remy.Storage;
+import Remy.Task.TaskList;
+import Remy.Ui;
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class MarkCommandTest {
+
+    @Test
+    public void MarkCommand_missingIndex_throwChatbotException() {
+        assertThrows(ChatbotException.class, () -> new MarkCommand("mark"));
+    }
+
+    @Test
+    public void MarkCommand_invalidIndex_throwChatbotException() {
+        assertThrows(ChatbotException.class, () -> new MarkCommand("mark abc"));
+    }
+
+    @Test
+    public void MarkCommand_negativeIndex_throwChatbotException() {
```

Comment:
Great job here. The titles here are not too general and can be easily comprehended 👍🏻